app/[locale]/header.tsx

```diff
@@ -126,11 +127,41 @@ export default async function Header({ locale }: { locale: string }) {
               <Link href={`/${locale}/login`}>{text.login}</Link>
             </Button>
           </li>
-          <li className="lg:hidden">
-            <IoMenu
-              className="rounded-md bg-primary-700 p-1 text-shade-light hover:cursor-pointer"
-              size={40}
-            />
+          <li className="z-30  font-semibold lg:hidden">
```

Comment:
Why does this keep happening?
```suggestion
          <li className="z-30 font-semibold lg:hidden">
```

Reply:
ill be more careful next time, prettier doesnt remove those